setup.sh

```diff
@@ -405,6 +405,15 @@ log "Pushing initial cluster configuration"
 git add config.yaml footloose.yaml machines.yaml flux.yaml wks-controller.yaml
 
 git diff-index --quiet HEAD || git commit -m "Initial cluster configuration"
+
+# Confirm it works
+read -p "Would you like to push to origin? (y/n) " -n 1 -r
```

Comment:
Please print what the origin is including the branch.

Reply:
Thank you, sorry for the late reply.. I believe the following commits fix this.